intel/filterlists/updater.go

```diff
@@ -23,17 +23,19 @@ func tryListUpdate(ctx context.Context) error {
 
 	if err != nil {
 		if !isLoaded() {
-			module.Error(filterlistsDisabled, err.Error())
+			warnAboutDisabledFilterLists()
 		} else {
-			module.Warning(filterlistsUpdateFailed, err.Error())
+			module.Warning(
+				filterlistsUpdateFailed,
+				"Filter Lists Update Failed",
+				fmt.Sprintf("The filter lists system failed to process an update. Depending on the previous state, the filtering capabilities are now either impaired or not given. Refer to the error for more details: %s", err.Error()),
+			)
```

Comment:
I'm not sure if talking about the "previous state" this is too technical. Also, I would remove the "Refer to the error" part as that makes only sense when reading the source code and would just do "Error: "